### ISSUE TITLE
plugin's models translations not working

### ISSUE THREAD

**Issue by dziurawy** [rainlab/translate-plugin#207]
I have followed the documentation, implemented TranslatableModel, added $translatable array but nothing changed in the backend forms.. am i missing something?

**Comment by Glitchbone** [comment]
Same problem here. I had to explicitly declare the multilingual versions of the form components I want to use in fields.yaml. mltext to have a translatable text field for example.

**Comment by dziurawy** [comment]
@Glitchbone  can you post the example how to fix?

**Comment by Glitchbone** [comment]
For example :

```
fields:
  title:
    label: Title
    type: mltext
  body:
    label: Body
    type: mlricheditor
```

**Comment by dziurawy** [comment]
Thanks,

If this is required step, it should be added to documentation.

**Comment by Glitchbone** [comment]
I don't think it's a required step, it's a bug

**Comment by Glitchbone** [comment]
This bug was fixed in 1.2.8
